allennlp/training/trainer.py

```diff
@@ -166,6 +167,16 @@ def train(self) -> None:
                 metrics["loss"] = float(train_loss / batch_num)
                 description = self._description_from_metrics(metrics)
                 train_generator_tqdm.set_description(description)
+
+                batch_num_tot = num_training_batches * epoch + batch_num
```

Comment:
Can this be `total` instead of `tot`?  It's a lot easier to read without the shortening.

Reply:
Sure